### PR TITLE
lesson-1

### PR DESCRIPTION
**Pull request by Merildew**

### CODE REVIEW

File: triphouse/src/components/HotelsItem.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return(
+        <div className='homes-item' >
+          <img className="homes-img" src={imageUrl} alt={name} />
+          <a className="homes-link" href="#">{name}</a>
+          <span className="homes-place">{city}, {country}</span>
+        </div>
+      );
+}
+
+export default HotelsItem;
```
Comment (Dronom) [inline triphouse/src/components/HotelsItem.jsx]: store propTypes after the function. it is much cleaner and more convenient to read
image

File: triphouse/package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
     "react": "^17.0.2",
```
Comment (Dronom) [inline triphouse/package.json]: Why do you have two package.json in a project?

**Review by Dronom — COMMENTED** [review]
I like your folder structure. Nevertheless, reconfigure package.json so that the project can be run without being in the triphouse folder.

File: triphouse/package.json (hunk 2 above)
Comment (Merildew) [reply triphouse/package.json]: there was a logical explanation for that, but for me would be easier just to delete 1 package.json :))

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,23 @@
-.idea
```
Comment (Dronom) [inline .gitignore]: I don't think that this folder with files should be in the git

Comment (Merildew) [reply .gitignore]: didn't notice sorry

**Pull request merged**